templates/workspace_services/mysql/porter.yaml

```diff
@@ -65,6 +65,7 @@ outputs:
     type: string
     applyTo:
       - install
+      - upgrade
```

Comment:
In this bundle upgrade isn't implemented and doesn't have an output defined as `mysql_fqdn`. So leave this one off. I suggest we open another issue to implement upgrade on the bundle.